src/styles/index.scss

```diff
@@ -1,3 +1,234 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
```

Comment:
It seems you've started to import a URL but never finished it. Please remove this line if it's not being used, or complete the import statement if you intended to import an external resource.